install-ubuntu-mate.sh

```diff
@@ -75,4 +75,5 @@ log_success_msg "DeskPi PWM Fan Control and Safeshut Service installed successfu
 log_action_msg "System will reboot in 5 seconds to take effect." 
 sudo sync
 sleep 5 
```

Comment:
No need for a sleep since the reboot call is no longer being made.  I assume the original intent was that it gave a chance to Ctrl+C the script although 5 secs was a bit to short to be reasonable.

Reply:
you are right on this one too. Overall I want to rewrite the script and make it fit for multiple ubuntu versions, but I can verify it only on the LTS version. Perhaps I should not bother to adjust a script but add one I have tested, and make that one an example for DeskpiTeam. Still rebooting a linux box without admin consent and for no good reason is a windoze habit.

Reply:
Given the lack of responses from the DeskPi team it might be more useful to create a community version of the scripts and fan tools in a separate repository.